front-end/src/main/kotlin/org/jetbrains/kotlin/spec/viewer/SpecTestsViewer.kt

```diff
@@ -82,63 +100,92 @@ class SpecTestsViewer {
         val helperFilesPromises = mutableListOf<Promise<Map<String, Any>>>()
 
         helperNames.forEach { helperName ->
-            helperFilesPromises.add(SpecTestsLoader.loadHelperFile(helperName))
+            //todo fix  helperFilesPromises.add(SpecTestsLoader.loadHelperFile(helperName))
```

Comment:
Hm. How are helper files loaded now?

Reply:
helper files've not been loaded correctly before. Right now this files do not load at all. Going to fix this issue as a next task